build.ps1

```diff
@@ -565,6 +596,7 @@ switch -Exact ($Target) {
         #VSX v110
 		VSX v120
         VSX v140
+		VSX v141
```

Comment:
VS2017 is ABI (binary) compatible with VS2015. This change is mainly for those that only have VS2017 installed.